### PR TITLE
Launching Puluminaries blog post

### PR DESCRIPTION
**Pull request by mattstratton**
Signed-off-by: Matt Stratton <matt.stratton@gmail.com>

### CODE REVIEW

**Comment by mattstratton** [comment]
There is still some more words to add and the meta image to create, but wanted to get some eyes before I went too much further.

**Comment by aaronkao** [comment]
- More details on Komal's contribution especially Auto API would be great to call out. 
- Paul Hicks image is blurry
- I feel like Simen's commentary is rather lackluster. We should have at least naother sentence of what he has done for us. 
- Is there anything else that Ringo has done beyond Pulumiverse? 

How does someone apply to be a Puluminary? Do we want them to send an email to da@pulumi.com?

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+---
+
+Pulumi is more than a way to build, deploy, and manage your infrastructure and cloud applications. Pulumi is also a strong and vibrant community. We are very excited to announce and showcase our new program of community champions, the Puluminaries!
+
+<!--more-->
+<img src="puluminaries.png" width="500" height="500" alt="Engin Diri"/>
+
+The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure. The program is open to all members of the community, and we welcome anyone to join the program.
+
+While the Puluminaries have been active in the community for quite some time, we have officially launched the program at [PulumiUP](https://pulumi.com/pulumi-up) in 2022, and it is our pleasure to introduce the first cohort of members to the program!
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:43]: While the Puluminaries themselves have been active... I think sounds better and refers to them not as a corporate entity but rather individuals. 

i don't think you need the "have" in "we have officially launched".

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Pulumi is more than a way to build, deploy, and manage your infrastructure and cloud applications. Pulumi is also a strong and vibrant community. We are very excited to announce and showcase our new program of community champions, the Puluminaries!
+
+<!--more-->
+<img src="puluminaries.png" width="500" height="500" alt="Engin Diri"/>
+
+The Puluminaries program was created to recognize members of the Pulumi community who are experts and leaders in the field of programming and infrastructure. The program is open to all members of the community, and we welcome anyone to join the program.
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:41]: I think its more than just being an expert and leader. Its people who give back to the community. They help others get started, they give feedback that make our roadmap even better, they educate people through content or speaking.

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    - /puluminaries/
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Pulumi is more than a way to build, deploy, and manage your infrastructure and cloud applications. Pulumi is also a strong and vibrant community. We are very excited to announce and showcase our new program of community champions, the Puluminaries!
+
+<!--more-->
+<img src="puluminaries.png" width="500" height="500" alt="Engin Diri"/>
```
Comment (stack72) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:39]: Engins pic twice?

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+<a data-track="twitter" href="https://twitter.com/blackgirlbytes"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/blackgirlbytes"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://www.linkedin.com/in/rizel-bobb-semple/"><i class="fab fa-linkedin"></i></a>
+
+Rizel brings her remarkable talent for guidance and teaching to the Pulumi community. She has participated in many Pulumi events, including [Cloud Engineering Summit](https://www.pulumi.com/resources/from-code-to-cloud/), and [appearing on the PulumiVision livestream](https://youtu.be/2uQEIYuJBZ4).
+
+### Simen A. W. Olsen
+
+<img src="simen-olsen.png" width="200" height="200" alt="Simen W Olsen"/>
+<a data-track="github" href="https://github.com/cobraz"><i class="fab fa-github"></i></a>
+
+Another one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/), Simen has a great passion for technology communities, but especially for the Pulumi community. Simen has also contributed code to the [Pulumi GitHub Action](https://github.com/pulumi/actions)!
```
Comment (stack72) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:101]: I'd change this to Simen was instrumental on the GitHub Action rewrite - he largely architected that and in his own free time

Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:101]: thank you! I didn't have context on what he did for it, and I meant to ask you about that. This is great and I'll update to include this.

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 3 above)
Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:39]: whoops. alt tag issue.

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+<a data-track="twitter" href="https://twitter.com/usrbinkat"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/usrbinkat"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/usrbinkat"><i class="fab fa-linkedin"></i></a>
+
+Kat has been a helpful member of the Pulumi community - not only providing feedback on real-world use of Pulumi, but also sharing her knowledge with [workshops and webinars](https://konghq.com/webinars/devmyops-deploy-kong-with-pulumi?utm_souce=pulumi&utm_medium=pulumi)
+
+### Komal Ali
+
+<img src="komal-ali.png" width="200" height="200" alt="Komal Ali"/>
+<a data-track="twitter" href="https://twitter.com/zwitkali"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/komalali"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/komal-ali"><i class="fab fa-linkedin"></i></a>
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers - especially bringing to life Pulumi's Automation API!
```
Comment (stack72) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:73]: Whilst at Pulumi, Komal was ever present in making our Python SDK better and championing the automation api.

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
+
+### Ringo De Smet
+
+<img src="ringo-smet.png" width="200" height="200" alt="Ringo De Smet"/>
+<a data-track="twitter" href="https://twitter.com/ringods"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/ringods"><i class="fab fa-github"></i></a>
+
+Ringo is one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/) - a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials. Ringo (along with fellow Puluminary Simen A. W. Olsen) have brought their vision for expanding the Pulumi community to life with the creating of this community-operated place for anyone to communicate, contribute, and collaborate with other members of the Pulumi community and build abstractions and SDKs that can be shared and used by anyone. Ringo also created the [Pulumi Resource Type for Concourse](https://github.com/ringods/pulumi-resource).
```
Comment (stack72) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:87]: I'd personally mention here how Ringo is ever present in every community he's part of as he championed Terrafarm too

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+Kat has been a helpful member of the Pulumi community - not only providing feedback on real-world use of Pulumi, but also sharing her knowledge with [workshops and webinars](https://konghq.com/webinars/devmyops-deploy-kong-with-pulumi?utm_souce=pulumi&utm_medium=pulumi)
+
+### Komal Ali
+
+<img src="komal-ali.png" width="200" height="200" alt="Komal Ali"/>
+<a data-track="twitter" href="https://twitter.com/zwitkali"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/komalali"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://linkedin.com/in/komal-ali"><i class="fab fa-linkedin"></i></a>
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers -  they were ever-present in making our Python SDK better and championing [Automation API](https://www.pulumi.com/docs/guides/automation-api/)!
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:73]: Didn't Komal just straight up build Auto API? Championing seems like a weak term for what was a much greater accomplishment.

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+### Paul Hicks
+
+<img src="paul-hicks.png" width="200" height="200" alt="Paul Hicks"/>
+<a data-track="github" href="https://github.com/tenwit"><i class="fab fa-github"></i></a>
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:80]: Prolific helper sounds light. I think we want to give a sense of the impressive knowledge that he brings to the table. 

Why don't we say "Paul seems to have an answer for everything. He is ever present in the Community answering the simple getting started questions to more complex cloud architecture questions" or something like that and as along as that is actually true.

File: themes/default/content/blog/introducing-the-puluminaries/index.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+<a data-track="twitter" href="https://twitter.com/blackgirlbytes"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/blackgirlbytes"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://www.linkedin.com/in/rizel-bobb-semple/"><i class="fab fa-linkedin"></i></a>
+
+Rizel brings her remarkable talent for guidance and teaching to the Pulumi community. She has participated in many Pulumi events, including [Cloud Engineering Summit](https://www.pulumi.com/resources/from-code-to-cloud/), and [appearing on the PulumiVision livestream](https://youtu.be/2uQEIYuJBZ4).
+
+### Simen A. W. Olsen
+
+<img src="simen-olsen.png" width="200" height="200" alt="Simen W Olsen"/>
+<a data-track="github" href="https://github.com/cobraz"><i class="fab fa-github"></i></a>
+
+Another one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/), Simen has a great passion for technology communities, but especially for the Pulumi community. Simen was instrumental on the [Pulumi GitHub Action](https://github.com/pulumi/actions) rewrite - designing and architecing it in his free time.
```
Comment (aaronkao) [inline themes/default/content/blog/introducing-the-puluminaries/index.md:101]: "Simen has a great passion for technology communities, but especially for the Pulumi community." - this is a bit nebulous and a filler. Anything else great we can call out to? If not, that is fine. 

architecting also has a typo

Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:101]: The github action thing was the big one Paul Stack mentioned - for the communities bit it was more about how he had a lot of thoughts about how to construct the board for Pulumiverse.

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 7 above)
Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:73]: yeah, I had said something to that effect, but "championing" was what paul stack suggested from his experience?

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 8 above)
Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md]: I updated this and added your suggestions!

File: themes/default/content/blog/introducing-the-puluminaries/index.md (hunk 7 above)
Comment (mattstratton) [reply themes/default/content/blog/introducing-the-puluminaries/index.md:73]: I was using the terminology @stack72 gave me, but if it's accurate to say that they built it themselves that way, we can update. I think there is a way to get both in there though.

**Comment by mattstratton** [comment]
Moved into pulumi/pulumi-hugo-private/pull/57

**Pull request closed without merging**